frontend/Dockerfile

```diff
@@ -22,12 +22,20 @@ COPY yarn.lock .
 RUN yarn --frozen-lockfile
 
 FROM base AS production
-COPY --from=build /build/.next/ .next/
-COPY --from=build /deps/node_modules/ node_modules/
-COPY package.json .
-COPY next.config.js .
-COPY public/ public/
-RUN mkdir src/
-COPY src/server.js src/server.js
+
 ENV NODE_ENV=production
-CMD yarn start
+# Make files are readable execuatable bu the `node` user.
+ARG PERMISSION=0555
+COPY --chmod=${PERMISSION} --from=build /deps/node_modules/ node_modules/
```

Comment:
Does the chmod get applied recursively?

Reply:
?

Reply:
I couldn't find any mention in the docs but I running `stat` in the containers shows it's recursive
![image](https://user-images.githubusercontent.com/37152329/198573962-0f5f483f-7e4a-4635-894b-00610363c468.png)

Reply:
That says 755?

Reply:
yes this seems wrong

Reply:
It's different for the package.json file
![image](https://user-images.githubusercontent.com/37152329/198576747-2a1a0239-306b-46ee-bb9b-0e7a2e18af2f.png)

Reply:
These screenshots are really not helping me figure this out. Let me know when you have an answer.